crates/key-value-redis/src/store.rs

```diff
@@ -98,4 +108,110 @@ impl Store for RedisStore {
             .await
             .map_err(log_error)
     }
+
+    async fn get_many(&self, keys: Vec<String>) -> Result<Vec<(String, Option<Vec<u8>>)>, Error> {
+        self.connection
+            .lock()
+            .await
+            .keys(keys)
+            .await
+            .map_err(log_error)
+    }
+
+    async fn set_many(&self, key_values: Vec<(String, Vec<u8>)>) -> Result<(), Error> {
+        self.connection
+            .lock()
+            .await
+            .mset(&key_values)
+            .await
+            .map_err(log_error)
+    }
+
+    async fn delete_many(&self, keys: Vec<String>) -> Result<(), Error> {
+        self.connection
+            .lock()
+            .await
+            .del(keys)
+            .await
+            .map_err(log_error)
+    }
+
+    async fn increment(&self, key: String, delta: i64) -> Result<i64, Error> {
+        self.connection
+            .lock()
+            .await
+            .incr(key, delta)
+            .await
+            .map_err(log_error)
+    }
```

Comment:
Heh, I look at these implementations compared to the others, and it becomes _very_ clear what the `atomics` and `batch` interfaces were modelled on...!